doc/nrf/libraries/others/hw_unique_key.rst

```diff
@@ -78,7 +78,7 @@ Additionally, you can enable the :kconfig:option:`CONFIG_HW_UNIQUE_KEY_RANDOM` o
 See :ref:`configure_application` for information on how to enable the required configuration options.
 
 You can then use the HUKs through the APIs in the :ref:`CC3xx platform libraries<nrf_cc3xx_platform_readme>`.
-You can also derive a key using :c:func:`hw_unique_key_derive_key`.
+You can also enable the :kconfig:option:`READ_HW_UNIQUE_KEY` option to derive a key using :c:func:`hw_unique_key_derive_key`.
```

Comment:
So this is not exactly accurate I think. Since this Kconfig option will not change anything in the code, it will just enforce some requires is how I see it.  i think that it has value to enforce this in code maybe? So basically ifdef hw_unique_key_derive_key  (and any functions that it calls) with the new Kconfig.

Reply:
Ok yeah wasn't sure if I should go that strict, but sounds good for me